vnext/local-cli/generator-windows/templates/cs/proj/MyApp.csproj

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="16.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\packages\Microsoft.UI.Xaml.2.3.191129002\build\Microsoft.UI.Xaml.props" Condition="Exists('..\packages\Microsoft.UI.Xaml.2.3.191129002\build\Microsoft.UI.Xaml.props')" />
```

Comment:
move to props file

Reply:
this cannot be moved in order to support proper updating.